concurrent/actor/Actor.java

```diff
@@ -69,26 +63,21 @@ private Actor(EventLoopGroup eventLoopGroup) {
 
     public void tell(Consumer<STATE> job) {
         assert state != null : ERROR_STATE_IS_NULL;
-        // assign it to a variable to suppress @CheckReturnValue deliberately
-        Promise<Object> compute = Promise.compute(eventLoop, () -> {
-            job.accept(state);
-            return null;
-        });
+        eventLoop.submit(() -> job.accept(state));
     }
 
     @CheckReturnValue
-    public Promise<Void> order(Consumer<STATE> job) {
```

Comment:
Do we want to remove `order` interface now? Then we will have to use a `Consumer` rather than `Function`.

Reply:
Are we still doing it in a lot of place in cluster?

Reply:
Yep we do use it in a bunch of places.

Reply:
Right ok, I thought we no longer use them as we've removed a bunch of actors already.

Reply:
How many usages still persist? If there's less than 5 I think we can just delete this function. I think it's convenient to have but doesn't sit well in the API

Reply:
We got about 20-ish of usages, though most of them are in `RaftTest`.

Reply:
Ok, let's keep the API then.